source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stompy.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Configuration for Stompy."""
+
+import omni.isaac.lab.sim as sim_utils
+from omni.isaac.lab.actuators import ImplicitActuatorCfg
+from omni.isaac.lab.assets.articulation import ArticulationCfg
+
+STOMPY_CFG = ArticulationCfg(
+    # Spawn stompy from URDF
+    spawn=sim_utils.UsdFileCfg(
+        usd_path="/home/dpsh/isaacs_sim/sim/sim/stompy/robot_fixed/robot_fixed.usd",
```

Comment:
shouldn't we use `source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/config/stompy/stompy/robot_fixed/robot_fixed.usd`?

Reply:
you're right i forgot to git add that file in the cleanup